src/containers/Permission.jsx

```diff
@@ -73,20 +74,18 @@ const Permission = ({ match }) => {
           {t('Permissions.failedRequest')}
         </Typography>
       ) : (
-        <div>
-          <IconButton
-            className="u-mr-half"
-            href={`#/permissions/${appName.toLowerCase()}`}
-          >
-            <Icon icon={PreviousIcon} size={16} />
-          </IconButton>
-
+        <React.Fragment>
+          <a href={`#${routes.appList}/${appName.toLowerCase()}`}>
+            <IconButton className="u-mr-half">
+              <Icon icon={PreviousIcon} size={16} />
+            </IconButton>
+          </a>
```

Comment:
It's weird to have an `<a>`. 

Can't you use https://docs.cozy.io/cozy-ui/react/#!/Buttons with icons + href props ? 🤔